examples/src/main/java/alluxio/examples/keyvalue/KeyValueStoreOperations.java

```diff
@@ -132,6 +132,7 @@ public static void main(String[] args) throws Exception {
       System.exit(-1);
     }
 
-    Utils.runExample(new KeyValueStoreOperations(new AlluxioURI(args[0])));
+    boolean result = CliUtils.runExample(new KeyValueStoreOperations(new AlluxioURI(args[0])));
```

Comment:
the "run and exit" pattern shows up repeatedly in the code base and it might make sense to add a utility function for it to `CliUtils`